src/doc/unstable-book/src/language-features/raw-dylib.md

```diff
@@ -26,9 +26,9 @@ fn main() {
 
 ## Limitations
 
-Currently, this feature is only supported on `-windows-msvc` targets.  Non-Windows platforms don't have import
-libraries, and an incompatibility between LLVM and the BFD linker means that it is not currently supported on
-`-windows-gnu` targets.
+This feature is unstable for the `x86` architecture, and stable for all other architectures.
```

Comment:
Since we aren't completely stabilizing the feature at this time, I think it makes sense to leave this page in the unstable-book but we should also file a PR against https://github.com/rust-lang/reference documenting the feature.

Reply:
I opened the PR to the reference a while ago: https://github.com/rust-lang/reference/pull/1244

Reply:
Oh, perfect! Thanks